<?xml version="1.0"?>
<launch>
  <master auto="start"/>

<!--- load robot's nodes -->
<include file="$(find robot)/launch/robot.launch"/>

<node pkg="random_walk" type="random_walk" name="random_walk" output="screen">
		<param name="max_linear_speed" value="0.25" />
		<param name="max_angular_speed" value="0.75" />
		<param name="safe_dist_th" value="0.75" />
		<param name="detour_obst_th" value="0.60" />
		<param name="stop_dist_th" value="0.40" />
		<param name="diff_th" value="0.1" />
		<param name="verbose" value="0" />
		<param name="useLRF" value="1" />
</node>

</launch>
